package.json

```diff
@@ -13,11 +13,14 @@
   "engines": {
     "node": ">=8"
   },
+  "resolutions": {
+    "simple-html-tokenizer": "0.5.7"
+  },
   "dependencies": {
     "@angular/compiler": "8.2.13",
     "@babel/code-frame": "7.5.5",
     "@babel/parser": "7.7.0",
-    "@glimmer/syntax": "0.41.0",
+    "@glimmer/syntax": "0.43.0",
```

Comment:
Would you mind updating the title to match this change?

Reply:
was trying other versions, its broken anyway,